### ISSUE TITLE
Channel attention (squeeze and excite) produces cryptic export with various extra operations

### ISSUE THREAD

**Issue by Artoriuz** [enpasos/jax2onnx#169]
When exporting the following residual block:

```python
class ResBlock(nnx.Module):
    def __init__(self, *, rngs: nnx.Rngs):
        self.conv0 = nnx.Conv(filters, filters, kernel_size=kernel_size, rngs=rngs)
        self.conv1 = nnx.Conv(filters, filters, kernel_size=kernel_size, rngs=rngs)
        self.conv2 = nnx.Conv(filters, filters // 8, kernel_size=(1, 1), rngs=rngs)
        self.conv3 = nnx.Conv(filters // 8, filters, kernel_size=(1, 1), rngs=rngs)

    def __call__(self, input):
        x = nnx.silu(self.conv0(input))
        f = self.conv1(x)
        x = jnp.mean(f, axis=(1,2), keepdims=True)
        x = nnx.silu(self.conv2(x))
        x = nnx.sigmoid(self.conv3(x))
        x = (x * f) + input
        return x
```

`jnp.mean(f, axis=(1,2), keepdims=True)` gets converted to this:

<img width="573" height="837" alt="Image" src="https://github.com/user-attachments/assets/f58c8e78-a85e-4b2e-9037-a6cb1a24ac9e" />

This entire block could be replaced with this single ONNX operation: https://onnx.ai/onnx/operators/onnx__ReduceMean.html
